### PR TITLE
Screenerform changes

### PR DESCRIPTION
**Pull request by sandeepsuri**

### CODE REVIEW

**Review by jas-lamba — CHANGES_REQUESTED** [review]
you don't need 
`$this->load->helper('url');
 $this->load->helper('form');
 $this->load->library('form_validation');`
 
 this is already autoloaded

File: emr/application/controllers/Screeners.php [hunk 1]
```diff
@@ -140,58 +140,108 @@ public function added_time() {
       $data['last_name'] = $this->ion_auth->user()->row()->last_name;
       $data['employeeid'] = $this->ion_auth->user()->row()->employeeid;
       $data['email'] = $this->ion_auth->user()->row()->email;
-
```
Comment (sandeepsuri) [inline emr/application/controllers/Screeners.php:143]: I removed those three lines of code!

Comment (jas2996) [reply emr/application/controllers/Screeners.php:143]: and did it work as expected?

File: emr/application/controllers/Screeners.php [hunk 2]
```diff
@@ -140,58 +140,108 @@ public function added_time() {
...
         
         $morn = array(
           'employeeid' => $data['employeeid'],
           'first_name' => $data['first_name'],
           'last_name' => $data['last_name'],
-          'start' => $morn_times[$i],
+          'start' => $morn_time[$i],
           'end' => $end_date
         );
         $this->db->insert('availability', $morn);
```
Comment (jas2996) [inline emr/application/controllers/Screeners.php]: you shouldn't do database actions in the controller, the model should be used for that instead

**Review by jas2996 — CHANGES_REQUESTED** [review]
make changes to how data is inserted into the database

File: emr/application/controllers/Screeners.php (hunk 1 above)
Comment (sandeepsuri) [reply emr/application/controllers/Screeners.php:143]: Yes, it was working for me without showing the error

File: emr/application/controllers/Screeners.php (hunk 2 above)
Comment (sandeepsuri) [reply emr/application/controllers/Screeners.php]: Just changed this, now its inserting into the db through the Availability_model

File: emr/application/controllers/Screeners.php (hunk 1 above)
Comment (sandeepsuri) [reply emr/application/controllers/Screeners.php:143]: yes it did

File: emr/application/controllers/Screeners.php (hunk 2 above)
Comment (jas2996) [reply emr/application/controllers/Screeners.php]: Sounds good

File: emr/application/controllers/Screeners.php [hunk 3]
```diff
@@ -176,7 +176,8 @@ public function added_time() {
           'start' => $morn_time[$i],
           'end' => $end_date
         );
-        $this->db->insert('availability', $morn);
+          $this->load->model('Availability_model');
```
Comment (jas2996) [inline emr/application/controllers/Screeners.php]: You should do this in the autoload.php file on line 137 instead of loading the model each time

**Review by jas2996 — CHANGES_REQUESTED** [review]
See my comment on autoloading the Availability_model

File: emr/application/controllers/Screeners.php (hunk 1 above)
Comment (jas2996) [reply emr/application/controllers/Screeners.php:143]: Sounds good!

File: emr/application/controllers/Screeners.php (hunk 3 above)
Comment (sandeepsuri) [reply emr/application/controllers/Screeners.php]: Done!

Comment (jas2996) [reply emr/application/controllers/Screeners.php]: Looks good!

File: emr/application/controllers/Screeners.php [hunk 4]
```diff
@@ -174,9 +174,9 @@ public function added_time() {
           'first_name' => $data['first_name'],
           'last_name' => $data['last_name'],
           'start' => $morn_time[$i],
-          'end' => $end_date
+          'end' => $end_date,
+          'shift_type' => 'Morning'
```
Comment (jas2996) [inline emr/application/controllers/Screeners.php]: lowercase shift_type value

**Review by jas2996 — CHANGES_REQUESTED** [review]
Its good practice to have data without any uppercase characters

File: emr/application/controllers/Screeners.php (hunk 4 above)
Comment (sandeepsuri) [reply emr/application/controllers/Screeners.php]: done!

Comment (jas2996) [reply emr/application/controllers/Screeners.php]: great!

**Comment by jas2996** [comment]
Ok this looks good to me now! Soon as you confirm that you have thoroughly tested this I'll merge this branch into master.

**Pull request merged**